src/styles/blocks/page.css

```diff
@@ -0,0 +1,20 @@
+html {
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+:root {
+  --blue: #00acdc;
+}
+
+.container__card {
+  display: flex;
+  flex-direction: column;
+  padding: 0 16px;
+}
+
+.card {
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 198px;
```

Comment:
use style `box-sizing: border-box` and define `width` with 200px as described in requirements